apps/platform-app/package.json

```diff
@@ -13,10 +13,13 @@
     "prisma:seed": "yarn prisma db seed",
     "postinstall": "prisma generate",
     "vercel-build": "prisma generate && prisma migrate deploy && next build",
-    "cypress:open": "cypress open"
+    "cypress:open": "cypress open",
+    "prisma:generate": "prisma generate --schema=$PRISMA_SCHEMA_PATH",
```

Comment:
**question:** Why did you need these?

Reply:
When I used command "npx prisma generate" I received an error that the schema.prisma couldn't be found. So that that was my first try to solve the case of changed location. I fixed it later with new instructions and forgot to remove it.